src/main/java/in/twizmwaz/cardinal/module/modules/scoreboard/ScoreboardModule.java

```diff
@@ -171,8 +175,16 @@ public void onScoreUpdate(ScoreUpdateEvent event) {
         updateTeamScore();
     }
 
+    public void updateObs(){
```

Comment:
Tiny formatting issue, add a space inbetween the updateObs constructor and the squiggly bracket.